src/libraries/Microsoft.PowerFx.Connectors/ArgumentMapper.cs

```diff
@@ -82,6 +98,35 @@ public ArgumentMapper(IEnumerable<OpenApiParameter> parameters)
                 }
             }
 
+            if (requestBody != null)
+            {
+                var bodyName = requestBody.GetBodyName() ?? BodyParameter;
+                FxOpenApiParameter bodyParameter;
+
+                if (requestBody.Content != null && requestBody.Content.Any())
+                {
+                    var schema = requestBody.Content.First().Value.Schema;                    
```

Comment:
>Content.First() [](http://example.com/codeflow?start=45&length=15)

Should we throw NotSupported unless Length=1?

Reply:
Nope
There are usually multiple and from what I've observed, the ".Value.Schema" is the same for all
Not sure if that's a universal rule though